dhlab/api/dhlab_api.py

```diff
@@ -68,21 +72,21 @@ def pos_from_urn(
         Check which models are available with :func:`show_spacy_models`
     :return: Dataframe with annotations and their frequencies
     """
-    params = {"urn": urn, "model": model, "start_page": start_page, "to_page": to_page}
-    r = requests.get(f"{BASE_URL}/pos_urn", params=params)
-    df = pd.read_json(r.json())
-    return df
+    resp = api_get(
+        f"{BASE_URL}/pos_urn",
+        params={"urn": urn, "model": model, "start_page": start_page, "to_page": to_page}
+    )
+
+    return pd.read_json(r.json())
 
 
 def show_spacy_models() -> List:
     """Show available SpaCy model names."""
-    try:
-        r = requests.get(f"{BASE_URL}/ner_models")
-        res = r.json()
-    except:
-        print("Server-request gikk ikke gjennom. Kan ikke vise SpaCy-modellnavn.")
-        res = []
-    return res
+    resp = api_get(
+        f"{BASE_URL}/ner_models",
+    )
```

Comment:
This one can probably be on one line

```suggestion
    resp = api_get(f"{BASE_URL}/ner_models")
```